website/www/site/data/en/pillars.yaml

```
@@ -12,7 +12,13 @@
 
 - title: Unified
   body: Use a single programming model for both batch and streaming use cases.
-- title: Portable
-  body: Execute pipelines on multiple execution environments.
+  icon: icons/unified-icon.svg 
 - title: Extensible
   body: Write and share new SDKs, IO connectors, and transformation libraries.
+  icon: icons/extensive-icon.svg
```

```suggestion
  icon: icons/extensible-icon.svg
```

This icon isn't rendering. I think this is the problem